src/pages/ui/main.tsx

```
@@ -8,10 +8,11 @@ import { CategorySelect } from '@features/restaurant';
 
 import { Category } from '@entities/restaurant';
 
-import { ErrorBoundary, RestaurantSkeleton } from '@shared/ui';
+import { GlobalErrorBoundary, LoadingState, RestaurantSkeleton } from '@shared/ui';
 
 export const MainPage = () => {
   const [category, setCategory] = useState<Category | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
```

ESLint

[error] 'setIsLoading' is assigned a value but never used.

(no-unused-vars)